pkg/csi/service/vanilla/controller.go

```diff
@@ -357,9 +356,8 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 	csiMigrationFeatureState := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration)
 	scParams, err := common.ParseStorageClassParams(ctx, req.Parameters, csiMigrationFeatureState)
 	if err != nil {
-		msg := fmt.Sprintf("Parsing storage class parameters failed with error: %+v", err)
-		log.Error(msg)
-		return nil, status.Errorf(codes.InvalidArgument, msg)
+		return nil, logger.LogNewErrorCodef(log, codes.InvalidArgument,
+			"Parsing storage class parameters failed with error: %+v", err)
```

Comment:
Can you change the first letter to lower case?
Parsing -> parsing

Reply:
The original code is not consistent. I probably will miss a lot of such cases.
Changing one anti-pattern a time is more efficient for me.

Anyway, I fixed it since you asked for it.